docs/backend/upgrading/version-specific-migration/upgrade-to-60.md

```diff
@@ -650,3 +650,8 @@ Plone 6.0 makes changes to two viewlet managers:
 One final change is that Plone 6.0 moves the `plone.footer` viewlet from `plone.app.layout/viewlets` to `plone.app.portlets`.
 The viewlet remains in manager `plone.portalfooter`.
 It renders the portlets from the `plone.footerportlets` portlet manager.
+
+## Boolean fields
+
+Since `zope.schema==6.1.0` all `zope.schema.Bool` fields need to have a `required=False` attribute,
```

Comment:
```suggestion
Since `zope.schema==6.1.0`, all `zope.schema.Bool` fields must have a `required=False` attribute.
This allows you to either tick or not tick the checkbox, submit the form, and process the field with either its value when ticked or `None` when unticked.
```